kerchunk/fits.py

```diff
@@ -62,7 +61,9 @@ def process_file(
         This allows you to supply an fsspec.implementations.reference.LazyReferenceMapper
         to write out parquet as the references get filled, or some other dictionary-like class
         to customise how references get stored
-
+    zarr_version: int
+        The desired zarr spec version to target (currently 2 or 3). The default
+        of None will use the default zarr version.
```

Comment:
"default" here means whatever version of zarr-python you have installed?

Reply:
Not exactly, it's whatever `zarr` itself sets as the default format. zarrv2 sets 2 as the default and zarrv3 sets 3 as the default, but strictly speaking it is up to `zarr`.